portfolio/src/main/webapp/script.js

```diff
@@ -55,8 +55,29 @@ const initialiseNavigation = () => {
   sectionSelector.addEventListener('change', function (e) {
     navigate(e.target.value);
   });
+
+  document.body.onscroll = onscroll;
 };
 
+const onscroll = () => {
+  const container = document.documentElement;
+  const scrollHeight = container.scrollHeight - container.clientHeight;
+  const scrollTop = container.scrollTop;
+
+  const percentage = Math.floor(scrollTop / scrollHeight * 100);
+  const section = Math.floor(percentage / 20);
+
+  // section == 5 is the comment section, which is not on the nav.
```

Comment:
Define a named const (something like COMMENT_SECTION); put this comment on the const definition and use that instead of the literal `5` in the code below.
If you can frame the section computations above around a single constant (replacing `/ 20` by something that depends on a constant), that would be even better. Maybe 5 is also the total number of sections?

Reply:
Added constants representing the referred sections and updated the logic to better represent the purpose of /20.
Indeed, I divided by 20 since there are 5 sections (underlying logic, 100 / NUMBER_OF_SECTIONS) 

Commit 162ab98

Reply:
This is awesome! Now move this comment line (`section == 5 ...`) to the definition of the constant.

Reply:
Updated comments 
b693bde